packages/components/input/Field.ts

```diff
@@ -251,6 +282,10 @@ export abstract class Field<T> extends Input<T> {
 				visibility: hidden;
 			}
 
+			:host(:focus) label, input:focus ~ label, :host([open]) label, input:not(:placeholder-shown) ~ label {
```

Comment:
Diesen Block verstehe ich garnicht. In der Zeile 225 ist der Default-Value schon `--mo-field-font-size`. Wieso überschreiben mit dem selben Wert?

Reply:
Damit das Label, wenn das über dem Text steht, nicht zu klein wird. Das war nämlich ein Problem, man konnte das Label nicht mehr gut lesen.

Reply:
> Damit das Label, wenn das über dem Text steht, nicht zu klein wird. Das war nämlich ein Problem, man konnte das Label nicht mehr gut lesen.

Das beantwortet nicht die Frage!
Ich sehe was diese Zeile machen sollte, aber was ich nicht nachvollziehen kann ist, an welcher Stelle überhaupt die `font-size` geändert wurde, damit die Notwendigkeit hier besteht die nochmal anzupassen?

Oben steht, dass `label` immer `font-size: var(--mo-field-font-size)` hat.

Reply:
Ich habe es noch mal geprüft. Das scheint doch noch irgendein überbleibsel zu sein. Ich habe es entfernt. Es gab mal so ein Problem und ich dachte vorhin das wäre die Stelle mit der es gefixt wurde.